src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 0.3s;
+    transition-property: transform, color;
```

You can also combine it
```suggestion
    transition: transform 0.3s color 0.3s;
```

---

changed